SingularityUI/app/controllers/TaskDetail.coffee

```diff
@@ -135,6 +172,17 @@ class TaskDetailController extends Controller
         @models.task.fetch()
             .done =>
                 @fetchResourceUsage() if @models.task.get('isStillRunning')
+            .success =>
+                requestId = @models.task.attributes.task.taskRequest.request.id
+                deployId = @models.task.attributes.task.taskRequest.deploy.id
+                deployInfo = new DeployDetails
+                  deployId: deployId
+                  requestId: requestId
+                deployInfo.fetch().success =>
```

Comment:
technically we could only make this call only if it's needed (i.e. if the task is scheduled + currently running)